include/osquery/external_hash.h

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
I know there are some changes in the works. ;) But I wanted to weigh in quickly, I don't think an "external hash" wrapper is needed. We should encourage hash callsites to use the SQL query class.